.circleci/config.yml

```diff
@@ -856,6 +870,15 @@ workflows:
     jobs:
       - release-train:
           xcode_version: '14.2.0'
+
+  daily-loadshedder-integration-tests:
+    when:
+      and:
+        - equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
+        - equal: [ "load_shedder_integration_tests", << pipeline.schedule.name >> ]
+    jobs:
+      - loadshedder-integration-tests:
+          xcode_version: '14.2.0'
```

Comment:
For reference, here's the CircleCI configuration:
<img width="682" alt="Screenshot 2023-03-30 at 12 17 26 PM" src="https://user-images.githubusercontent.com/3922667/228884235-f7ccbd04-8814-454c-83bd-3e901564c126.png">